libraries/chain/asset_evaluator.cpp

```diff
@@ -378,54 +378,55 @@ void check_children_of_bitasset(database& d, const asset_update_bitasset_operati
       return;
 
    // loop through all assets that have this asset as a backing asset
-   const auto& idx = d.get_index_type<asset_index>().indices().get<by_type>();
-
-   for( auto itr = idx.lower_bound(true); itr != idx.end(); ++itr )
-   {
-      const auto& child = *itr;
-      if ( child.bitasset_data(d).options.short_backing_asset == op.asset_to_update )
-      {
-         if ( after_hf_922_931 )
+   const auto& idx = d.get_index_type<graphene::chain::asset_bitasset_data_index>()
+         .indices()
+         .get<by_short_backing_asset>();
+   auto backed_range = idx.equal_range(op.asset_to_update);
+   std::for_each( backed_range.first, backed_range.second,
+         [after_hf_922_931, &new_backing_asset, &d, &op](const asset_bitasset_data_object& bitasset_data)
```

Comment:
I guess if change this indentation to smaller, the whole diff will look much better.

Reply:
Hint: diff -w (or "hide whitespace" in diff settings on github)

Reply:
Thanks. That does the trick.